fontFeatures/feeLib/Anchors.py

```diff
@@ -49,51 +49,83 @@
 
 """
 
+from . import HelperTransformer
```

Comment:
I know that in terms of the parser, a "helper transformer" is what this is; but from our perspective it's something else. A FEE verb, maybe? I think we need a more descriptive or at least FEE-friendly name.